src/reanimated2/Jest.js

```diff
@@ -0,0 +1,163 @@
+export let withReanimatedTimer;
+export let moveAnimationByTime;
+export let moveAnimationByFrame;
+export let getAnimatedStyle;
+
+export const SetUpTests = (userConfig = {}) => {
+  const expect = require('expect');
+  const config = {
+    fps: 60,
+    ...userConfig,
+  };
+
+  global.performance = {
+    now: () => Date.now(),
+  };
```

Comment:
Why do we need this global? Could you update the PR description?

Reply:
good point, actually, I don't need a global value to this. I replaced this with just `Date.now()`